service/service_test.go

```diff
@@ -425,8 +425,8 @@ func newNopConfig() Config {
 					Initial:    100,
 					Thereafter: 100,
 				},
-				OutputPaths:       []string{"stderr"},
-				ErrorOutputPaths:  []string{"stderr"},
```

Comment:
It's not obvious to me how this is related. Do you mind clarifying for me?

Reply:
Tests fail if we try to sync, because /dev/stderr is not available, most likely because the process doesn’t run with tty. Removing the output fixed the issue.

Reply:
So that was a bit painful... going to make a bigger comment on the PR itself with a follow up.